domains/raghavyuva.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "raghavyuva",
+        "email": "vikramnbhat15@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "185.199.108.153"
```

Comment:
I think you meant to use an `A` record for this?

Reply:
CNAME field has been updated,kindly check it out